Kong/docs.konghq.com
# chore: Update plugin submodule

### Description

Chore PR to update the plugins submodule for 3.5 one more time.

### Testing instructions

Make sure updates appear, mainly:
https://deploy-preview-6468--kongdocs.netlify.app/hub/kong-inc/opentelemetry/configuration/#config-resource_attributes

https://deploy-preview-6468--kongdocs.netlify.app/hub/kong-inc/acl/configuration/#config

### Checklist
